pkg/server/router/manifest.go

```diff
@@ -182,3 +185,150 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 
 	return framework.Respond(ctx, w, nil, http.StatusOK)
 }
+
+// Application
+type CreateApplicationRequest struct {
+	ManifestID             string                              `json:"manifestId" validate:"required"`
+	PresentationSubmission exchangensdk.PresentationSubmission `json:"presentationSubmission" validate:"required"`
+}
+
+func (c CreateApplicationRequest) ToServiceRequest() manifest.CreateApplicationRequest {
+	return manifest.CreateApplicationRequest{
+		PresentationSubmission: c.PresentationSubmission,
+		ManifestID:             c.ManifestID,
+	}
+}
+
+type CreateApplicationResponse struct {
+	Application applicationsdk.CredentialApplication `json:"application"`
+}
+
+// CreateApplication godoc
+// @Summary      Create application
+// @Description  Create application
+// @Tags         ApplicationAPI
+// @Accept       json
+// @Produce      json
+// @Param        request  body      CreateApplicationRequest  true  "request body"
+// @Success      201      {object}  CreateApplicationResponse
+// @Failure      400      {string}  string  "Bad request"
+// @Failure      500      {string}  string  "Internal server error"
+// @Router       /v1/manifests/applications [put]
+func (ar ManifestRouter) CreateApplication(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
```

Comment:
is this endpoint creating an application or accepting a created application?
meaning, should it instead be 'submitapplication'?

Reply:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/5314059/190028223-97324a87-47e9-4b00-9afb-923fd115eb76.png">
Right, I actually forgot to change the name to ApplyForCredential as per SIP2.

I think SubmitApplication makes a lot more sense though, I'll change it to this